tests/queries/0_stateless/02674_null_default_structure.sql

```diff
@@ -0,0 +1,2 @@
+SELECT * FROM null();
+DESCRIBE null();
```

Comment:
Let's also add `insert into table function  null() select 1, 'str'` to make sure `structure_hint` is preferred over default structure.